airflow/providers/cncf/kubernetes/utils/pod_manager.py

```diff
@@ -256,7 +293,7 @@ def consume_logs(*, since_time: Optional[DateTime] = None, follow: bool = True)
                 time.sleep(1)
 
     def await_container_completion(self, pod: V1Pod, container_name: str) -> None:
-        while not self.container_is_running(pod=pod, container_name=container_name):
+        while self.container_is_running(pod=pod, container_name=container_name):
```

Comment:
this looks like it is unrelated to your main change.  could you please raise a separatae PR for this change specifically?

Reply:
(it's less complicated, more obvious fix than the other part  so we can merge it faster and focus more cleanly on the other bit in this PR)

Reply:
ah, done here: https://github.com/apache/airflow/pull/23883/files now just need to rebase

Reply:
It looks green. @dstandish - do you approve :)?